figures/4.figure4/scripts/figure4.r

```diff
@@ -0,0 +1,903 @@
+suppressPackageStartupMessages(suppressWarnings(library(ggplot2)))
```

Comment:
Consider adding a comment of some kind to the top of this file to help describe the purpose.